diff --git a/node_modules/react-native-video/android/build.gradle b/node_modules/react-native-video/android/build.gradle
index 1c52da9..0a71638 100644
--- a/node_modules/react-native-video/android/build.gradle
+++ b/node_modules/react-native-video/android/build.gradle
@@ -7,7 +7,7 @@ def safeExtGet(prop, fallback) {
 android {
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
-
+    namespace 'com.brentvatne.react'
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
         targetSdkVersion safeExtGet('targetSdkVersion', 28)
diff --git a/node_modules/react-native-video/android/src/main/AndroidManifest.xml b/node_modules/react-native-video/android/src/main/AndroidManifest.xml
index 3535ad4..5fbaa4c 100644
--- a/node_modules/react-native-video/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-video/android/src/main/AndroidManifest.xml
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.brentvatne.react">
+          >
 </manifest>
