diff --git a/node_modules/react-native-safe-area-context/android/build.gradle b/node_modules/react-native-safe-area-context/android/build.gradle
index 736822d..66bbecc 100644
--- a/node_modules/react-native-safe-area-context/android/build.gradle
+++ b/node_modules/react-native-safe-area-context/android/build.gradle
@@ -47,7 +47,7 @@ if (isNewArchitectureEnabled()) {
 
 android {
     compileSdkVersion getExtOrDefault('compileSdkVersion', 30)
-
+    namespace 'com.th3rdwave.safeareacontext'
     // Used to override the NDK path/version on internal CI or by allowing
     // users to customize the NDK path/version from their root project (e.g. for M1 support)
     if (rootProject.hasProperty("ndkPath")) {
diff --git a/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml b/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml
index 479a914..3239c4f 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml
@@ -1,6 +1,6 @@
 
 <manifest
 	xmlns:android="http://schemas.android.com/apk/res/android"
-	package="com.th3rdwave.safeareacontext">
+	>
 
 </manifest>
