diff --git a/node_modules/@react-native-cookies/cookies/android/build.gradle b/node_modules/@react-native-cookies/cookies/android/build.gradle
index 13c7e43..693a601 100644
--- a/node_modules/@react-native-cookies/cookies/android/build.gradle
+++ b/node_modules/@react-native-cookies/cookies/android/build.gradle
@@ -42,6 +42,7 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
+    namespace 'com.reactnativecommunity.cookies'
     compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
     buildToolsVersion safeExtGet('buildToolsVersion', DEFAULT_BUILD_TOOLS_VERSION)
     defaultConfig {
diff --git a/node_modules/@react-native-cookies/cookies/android/src/main/AndroidManifest.xml b/node_modules/@react-native-cookies/cookies/android/src/main/AndroidManifest.xml
index a34a82b..6da2f3b 100644
--- a/node_modules/@react-native-cookies/cookies/android/src/main/AndroidManifest.xml
+++ b/node_modules/@react-native-cookies/cookies/android/src/main/AndroidManifest.xml
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.reactnativecommunity.cookies" >
+	>
 </manifest>
