diff --git a/node_modules/@react-native-community/datetimepicker/android/build.gradle b/node_modules/@react-native-community/datetimepicker/android/build.gradle
index f0c168e..ee7b6e5 100644
--- a/node_modules/@react-native-community/datetimepicker/android/build.gradle
+++ b/node_modules/@react-native-community/datetimepicker/android/build.gradle
@@ -8,7 +8,7 @@ def safeExtGet(prop, fallback) {
 android {
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
-
+    namespace 'com.reactcommunity.rndatetimepicker'
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 21)
         targetSdkVersion safeExtGet('targetSdkVersion', 28)
diff --git a/node_modules/@react-native-community/datetimepicker/android/src/main/AndroidManifest.xml b/node_modules/@react-native-community/datetimepicker/android/src/main/AndroidManifest.xml
index 714a934..c91836f 100644
--- a/node_modules/@react-native-community/datetimepicker/android/src/main/AndroidManifest.xml
+++ b/node_modules/@react-native-community/datetimepicker/android/src/main/AndroidManifest.xml
@@ -1,3 +1,3 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.reactcommunity.rndatetimepicker">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
   <application />
 </manifest>
