diff --git a/node_modules/react-native-webview/android/build.gradle b/node_modules/react-native-webview/android/build.gradle
index e6d0508..02ad354 100644
--- a/node_modules/react-native-webview/android/build.gradle
+++ b/node_modules/react-native-webview/android/build.gradle
@@ -59,7 +59,7 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
+  mavenCentral()
   google()
 
   def found = false
