diff --git a/node_modules/react-native-background-job/android/build.gradle b/node_modules/react-native-background-job/android/build.gradle
index ce16973..83d04df 100644
--- a/node_modules/react-native-background-job/android/build.gradle
+++ b/node_modules/react-native-background-job/android/build.gradle
@@ -42,7 +42,7 @@ repositories {
 }
 
 dependencies {
-  compile 'com.firebase:firebase-jobdispatcher:0.8.5'
-  compile 'com.facebook.react:react-native:+'
+  implementation 'com.firebase:firebase-jobdispatcher:0.8.5'
+  implementation 'com.facebook.react:react-native:+'
 }
 
