diff --git a/node_modules/realm/android/build.gradle b/node_modules/realm/android/build.gradle
index 19e9840..5b3d1e7 100644
--- a/node_modules/realm/android/build.gradle
+++ b/node_modules/realm/android/build.gradle
@@ -11,7 +11,7 @@ buildscript {
 allprojects {
     repositories {
         mavenLocal()
-        jcenter()
+        mavenCentral()
         maven {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
             url "$projectDir/../../react-native/android"
