diff --git a/node_modules/realm/react-native/android/build.gradle b/node_modules/realm/react-native/android/build.gradle
index 5bd196b..2ef4f99 100644
--- a/node_modules/realm/react-native/android/build.gradle
+++ b/node_modules/realm/react-native/android/build.gradle
@@ -24,7 +24,7 @@ apply plugin: 'com.android.library'
 android {
     compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : 28
     buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : '28.0.3'
-
+    namespace 'io.realm.react'
     defaultConfig {
         minSdkVersion rootProject.hasProperty('minSdkVersion') ? rootProject.minSdkVersion : 16
         targetSdkVersion rootProject.hasProperty('targetSdkVersion') ? rootProject.targetSdkVersion : 28
diff --git a/node_modules/realm/react-native/android/src/main/AndroidManifest.xml b/node_modules/realm/react-native/android/src/main/AndroidManifest.xml
index dbd8486..0fd6dad 100644
--- a/node_modules/realm/react-native/android/src/main/AndroidManifest.xml
+++ b/node_modules/realm/react-native/android/src/main/AndroidManifest.xml
@@ -1,3 +1,3 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.realm.react">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <application />
 </manifest>
